### ISSUE TITLE
On nethery wings (10438) (BB #953)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#953]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 02.06.2015 15:21:31 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/953

<hr>

On nethery wings (10438)

use your Phase Disruptor(29778) when you get in range of the Void Conduit

Bug: Void Conduit regens HP like every second that makes you not able to kill him with Phase Disruptor.

More info here -> [video](https://youtu.be/axHhoz_mwOw)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 22.11.2015 18:35:58 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/953/#comment-23498622

<hr>

Fix for this:

```
#!mysql
INSERT INTO `creature_template` (`entry`, `heroic_entry`, `KillCredit1`, `KillCredit2`, `modelid_A`, `modelid_A2`, `modelid_H`, `modelid_H2`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `minhealth`, `maxhealth`, `minmana`, `maxmana`, `armor`, `faction_A`, `faction_H`, `npcflag`, `speed`, `scale`, `rank`, `mindmg`, `maxdmg`, `dmgschool`, `attackpower`, `baseattacktime`, `rangeattacktime`, `unit_flags`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `class`, `race`, `minrangedmg`, `maxrangedmg`, `rangedattackpower`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `resistance1`, `resistance2`, `resistance3`, `resistance4`, `resistance5`, `resistance6`, `spell1`, `spell2`, `spell3`, `spell4`, `PetSpellDataId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `RacialLeader`, `RegenHealth`, `equipment_id`, `mechanic_immune_mask`, `flags_extra`, `ScriptName`) VALUES (20899, 0, 0, 0, 9832, 0, 9832, 0, 'Void Conduit', NULL, NULL, 0, 70, 70, 260000, 260000, 0, 0, 20, 14, 14, 0, 1.125, 1, 0, 202, 350, 0, 1932, 2000, 0, 4, 0, 0, 0, 0, 0, 0, 346.02, 481.06, 100, 10, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 1, 4, 0, 0, 0, 0, 2, '');


```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 30.11.2015 14:32:57 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/953/#comment-23685171

<hr>

It can be pushed to the repo, since it does work like it should. Issue can be also marked as resolved.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** desteny
**Original Date:** 01.12.2015 23:49:01 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/953/#comment-23730102

<hr>

Fix #953 thanks @Selphius

→ <<cset c3cbe61b24ff>>
